shell/platform/darwin/ios/framework/Source/FlutterAppDelegate.mm

```diff
@@ -15,6 +15,12 @@
 static NSString* const kRemoteNotificationCapabitiliy = @"remote-notification";
 static NSString* const kBackgroundFetchCapatibility = @"fetch";
 static NSString* const kRestorationStateAppModificationKey = @"mod-date";
+static NSString* const kApplicationDidReceiveRemoteNotificationFetchCompletionHandler =
```

Comment:
Should also do iOS 10 deprecated `application:didReceiveRemoteNotification:`
https://github.com/firebase/firebase-ios-sdk/blob/7eb1747e4b9be195ea9908b9da28b1d0f818afe3/GoogleUtilities/AppDelegateSwizzler/GULAppDelegateSwizzler.m#L115-L116

Reply:
I'd rather stick to fixing the linked issue.  I'm assuming it is correct and demonstrating that my changes are correct with tests.  Is there an issue for them not working on iOS 10?

Reply:
That's fine, we can do that one later.